### DISCUSSION TITLE
Duplicate Commits Showing in REST API Response

### DISCUSSION THREAD

**Discussion by ghost** [community/community#58435] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

I am getting duplicate commit in API Response. Also, on UI duplicate commits are being shown.

**Comment by DSDmark** [comment]
Hi @cosmoneo,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.
